src/routes/protected.route.js

```diff
@@ -12,7 +12,7 @@ class ProtectedRoute extends React.Component {
       localStorage.removeItem('appState');
     }
 
-    return isAuthenticated ? <Component {...props} /> : <Redirect to={{ pathname: '/get-started' }} />;
+    return isAuthenticated ? <Component {...props} /> : <Redirect to={{ pathname: '/map' }} />;
```

Comment:
you should throw a snackbar when people get booted to map page because of auth errors. maybe something like "You must have an account to visit this page." and a button in the snackbar that links to the account create page

Reply:
I asked product these questions...they said no need. A login and create button will be on the maps page.

Will raise to Matt and Product to see what they want with the transition

@crimsonsunset